package.json

```diff
@@ -11,12 +11,12 @@
     "connect-flash": "0.1.1",
     "cookie-parser": "^1.4.1",
     "debug": "0.7.4",
-    "express": "^4.13.4",
+    "express": "^4.14.0",
     "express-session": "^1.13.0",
-    "jade": "1.0.0",
+    "pug": "2.0.0-beta6",
```

Comment:
This will not work if you don't change app.js view engine for pug and renaming all views.